modules/200-operator-prometheus/monitoring/prometheus-rules/operator.yaml

```
@@ -84,9 +111,20 @@
       plk_grouped_by__d8_prometheus_operator_unavailable: "D8PrometheusOperatorUnavailable,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
       summary: The `prometheus-operator` Pod is NOT Running.
       description: |
-        The new `Prometheus`, `PrometheusRules`, `ServiceMonitor` settings cannot be applied in the cluster; however, all existing and configured components continue to operate correctly.
-        This problem will not affect alerting or monitoring in the short term (a few days).
+        As a result, new `Prometheus`, `PrometheusRules`, and `ServiceMonitor` configurations cannot be applied in the cluster.
+        However, all existing and configured components will continue to operate normally.
+        This problem will not affect alerting or monitoring in the short term (for a few days).
+
+        Troubleshooting steps:
+
+        1. Analyze the Deployment details:
+        
+           ```shell
+           kubectl -n d8-operator-prometheus describe deploy prometheus-operator
```

It is suggested to specify full commands everywhere, not aliases. 

```suggestion
           kubectl -n d8-operator-prometheus describe deployment prometheus-operator
```